### ISSUE TITLE
Windows - Installed DLC not being Detected after Quiting App

### ISSUE THREAD

**Issue by GumbyXGames** [sudachi-emu/sudachi#58]
Any DLC installed to NAND is not detected as installed after quitting app even though they are displayed on game list. When reinstalling them, Sudachi shows that they were overwritten.

![sudachi _ main-edd7896 4_11_2024 8_54_23 PM](https://github.com/sudachi-emu/sudachi/assets/82996624/397a7566-016e-4dca-a507-77af0c7e2bd6)

Build: vedd7896
OS: Win11 23H2

**Comment by GumbyXGames** [comment]
Screenshot

![Screenshot 2024-04-11 210130](https://github.com/sudachi-emu/sudachi/assets/82996624/c779b6a5-321e-4e99-adf6-26de62e9ced6)

**Comment by harrisonhacks** [comment]
I just posted a video on youtube that may help you out. Not trying to promote my channel or anything, just hoping to help out a fellow gamer.

**Comment by Nelk90** [comment]
I have the exact same issue.

**Comment by zzzribas** [comment]
Yes, im also getting this problem, if i install manually it works until i close the emulator and open again, i changed my directory of nand can it be it? Does it check when it starts for the default nand and after that unloads the addons or dlcs?

**Comment by YocksA** [comment]
> I just posted a video on youtube that may help you out. Not trying to promote my channel or anything, just hoping to help out a fellow gamer.

Where is it? aaa
